components-compose/src/main/java/uk/gov/hmrc/components/compose/organism/information/InformationMessageCardView.kt

```diff
@@ -0,0 +1,178 @@
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+import uk.gov.hmrc.components.compose.ui.theme.HmrcWhite
+import uk.gov.hmrc.components.compose.ui.theme.HmrcYellow
+
+enum class MessageType(val backgroundColor: Color, val headlineTint: Color, val iconColor: Color, val icon: Int) {
+    WARNING(HmrcYellow, HmrcAlwaysBlack, HmrcAlwaysBlack, R.drawable.components_ic_warning),
+    INFO(HmrcBlue, HmrcWhite, HmrcWhite, R.drawable.ic_info),
+    URGENT(HmrcRed, HmrcWhite, HmrcWhite, R.drawable.ic_info),
+    NOTICE(HmrcBlack, HmrcWhite, HmrcWhite, R.drawable.components_ic_warning)
+}
+
+@Composable
+fun InformationMessageCardView(
+    modifier: Modifier = Modifier,
+    headline: String,
+    headlineContentDescription: String = "",
+    text: String? = null,
+    messageType: MessageType,
+    buttons: List<InformationMessageButton> = emptyList()
+) {
+    val textColor: Color by remember { mutableStateOf(messageType.headlineTint) }
+    val iconColor: Color by remember { mutableStateOf(messageType.iconColor) }
+    val backgroundColor: Color by remember { mutableStateOf(messageType.backgroundColor) }
+    val icon: Int by remember { mutableIntStateOf(messageType.icon) }
+
+    HmrcCardView(modifier = modifier, customBackgroundColor = backgroundColor) {
+        Row(
+            modifier = modifier
+                .fillMaxWidth()
+                .enableTalkBackMergeAccessibility()
+                .padding(HmrcTheme.dimensions.hmrcSpacing16),
+        ) {
+            Image(
+                modifier = Modifier.size(HmrcTheme.dimensions.hmrcIconSize36),
+                painter = painterResource(id = icon),
+                contentDescription = "",
+                colorFilter = ColorFilter.tint(iconColor)
+            )
+            Spacer(modifier = Modifier.width(HmrcTheme.dimensions.hmrcSpacing8))
+            Text(
+                text = headline,
+                modifier = Modifier
+                    .weight(1f)
+                    .heightIn(HmrcTheme.dimensions.hmrcIconSize36)
+                    .wrapContentSize(align = Alignment.CenterStart)
+                    .semantics {
+                        this.contentDescription = headlineContentDescription
+                    },
+                style = HmrcTheme.typography.h6,
+                color = textColor
+            )
+        }
+        if (text != null) {
+            Row(
+                modifier = modifier
+                    .fillMaxWidth()
+                    .padding(
+                        start = HmrcTheme.dimensions.hmrcSpacing16,
+                        end = HmrcTheme.dimensions.hmrcSpacing16,
+                        bottom = HmrcTheme.dimensions.hmrcSpacing16
+                    )
+            ) {
+                Text(
+                    text = text,
+                    modifier = Modifier,
+                    style = HmrcTheme.typography.body,
+                    color = textColor
+                )
+            }
```

Comment:
Can we remove the parent Row here?

```suggestion
              Text(
                text = text,
                modifier = modifier
                    .fillMaxWidth()
                    .padding(
                        start = HmrcTheme.dimensions.hmrcSpacing16,
                        end = HmrcTheme.dimensions.hmrcSpacing16,
                        bottom = HmrcTheme.dimensions.hmrcSpacing16
                    ),
                style = HmrcTheme.typography.body,
                color = textColor
            )
```

Reply:
This breaks it